### ISSUE TITLE
The effect of `SpriteTest -> 33: Testing Sprite openGL Z vertex` is wrong

### ISSUE THREAD

**Issue by dumganhar** [cocos2d/cocos2d-x#16740]
- cocos2d-x version: 3.13.1
- devices test on: macOS
- developing environments
  - Xcode version: 8.0

Steps to Reproduce:
1.  cpp-tests -> Node: SpriteTest -> 33: Testing Sprite openGL Z vertex` is wrong

**Comment by dumganhar** [comment]
I have no idea of this issue. @ricardoquesada , could you please take a look at this issue?

**Comment by ricardoquesada** [comment]
yep. assigned to self. not sure what's the problem, but I've seen this bug before. I'll take a look at it later.
